astro/tests/fixtures/terraform

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+set -euo pipefail
+readonly VERSION='Terraform v0.12.7'
+
+# When Bash receives a signal for which a trap has been set
+# while waiting for a command to complete,
+# the trap will not be executed until the command completes.
+# http://tldp.org/LDP/Bash-Beginners-Guide/html/sect_12_02.html
+fake_work() {
+    local -r seconds="$1"
+    for ((i=0;i<=$(($seconds*10)); i++)) do
+        sleep 0.1
+    done
+}
+
+trap_with_arg() {
+    func="$1" ; shift
+    for sig ; do
+        trap "${func} ${sig}" "${sig}"
+    done
+}
+
+func_trap() {
```

Comment:
Maybe call it "display_trap_signal" or something. It took me some time to understand what trap_with_arg + trap_func are doing. Maybe if the names are more description that could help.

Also, why do we need to wait 2 seconds in the end?

Reply:
We're trying to simulate astro behavior here, although exactly 2 seconds are not needed here.